app/javascript/utilities/connect/index.js

```diff
@@ -1,2 +1,3 @@
 export * from './getUnopenedChannels';
 export * from './pusher';
+export * from './newMessageNotify';
```

Comment:
Refactor (blocking):  the `./newMessageNotify` import only exports a default function, I do not see the need to include this in this [barrel file](https://basarat.gitbook.io/typescript/main-1/barrel).

Consider having a named import in `'./newMessageNotify` if you want to have it in the barrel file.

Also, I do not see this being used anywhere, e.g. `import notifyUser from 'utilities/connect';`

Reply:
Done